docs/wiki/00_SERVER_INSTALL_AND_CONFIGURATION/02_Website_Configuration/README.md

```diff
@@ -1 +1,142 @@
-Web configuration.
\ No newline at end of file
+Web configuration.
+
+## Create front-end Users  
+
+The installation script adds admin user/password, associated with a dummy email address which should be changed. Additional users and permissions are added in the User Accounts page. For clarity and uniqueness, it is recommended to use an email address as username.  
+
+* To set/reset a user password, use the script [tools/resetpassword.php](https://github.com/aces/Loris/blob/master/tools/resetpassword.php)  
+
+
+## Create back-end accounts  
+
+Two types of back-end accounts are useful for administering Loris: MySQL accounts and Unix account.  
+
+It is good practice to create a new MySQL user for developer purposes (e.g. lorisDBadmin) to execute all back-end transactions, both in configuration and in day-to-day operations (i.e. not MySQL root). This account should be distinct from the MySQL root user as well as distinct from the lorisuser account which the install script created with limited permissions for executing all transactions coming from Loris' PHP code (i.e. from front-end users' activity).  
+
+```SQL
+GRANT ALTER, DROP, CREATE, UPDATE, INSERT, SELECT, DELETE, CREATE TEMPORARY TABLES, LOCK TABLES  on $dbname.* to 'lorisDBadmin'@'$dbhost' IDENTIFIED BY '$newpassword' WITH GRANT OPTION ;
+```  
+
+lorisadmin is the Loris administrator Unix account. Developers may wish to have their own individual accounts.  
+
+## Define Study Site
+
+Study sites or centres are defined in the psc table, with one row per Project Study Centre (PSC). LORIS' default schema defines the first psc (CenterID=1) as the Data Coordinating Center or #DCC#. This site is used for testing database features using dummy data and is assumed by the codebase to store non-study data, and so it not recommended to modify or use this site for registering real study data.  
+
+Populate additional sites using the following MySQL command-line :  
+``` SQL
+INSERT INTO psc (Name, Alias, MRI_alias, Study_site) VALUES ('Montreal','MTL','MTL','Y');
+```
+* The first study site will have its CenterID = 2  
+* There cannot be more than one site with the same name  
+
+## Configure study Variables  
```

Comment:
Hi @haoweiqiu  - much of this is already covered in the markdowns in the sections starting here: 
https://acesloris.readthedocs.io/en/latest/docs/wiki/01_STUDY_PARAMETERS_SETUP/01_Study_Variables/00_Introduction_to_Study_Variables.html#
Could you re-review your changes to compare what content has not already made it into the markdowns.

Reply:
Hi @christinerogers, thank you very much for pointing out this issue. I will check again my changes.